.github/workflows/tests.yml

```diff
@@ -18,6 +18,8 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - name: Redis Server in GitHub Actions
+      uses: supercharge/redis-github-action@1.4.0
```

Comment:
This should not be necessary. Integration test docker compose should spin up the redis server itself. Unit tests (tests run outside of the int test docker containers) should not rely on redis at all (mocked interfaces or similar should be used).